db/user.py

```diff
@@ -13,8 +13,11 @@ def create(musicbrainz_id):
 def get(id):
     """Get user with a specified ID (integer)."""
     with db.engine.connect() as connection:
-        result = connection.execute('SELECT id, created, musicbrainz_id FROM "user" WHERE id = %s',
-                       (id,))
+        result = connection.execute("""
```

Comment:
I've started using `text` for multiline queries, which allows us to add placeholders with `:id` and have the second parameter to `execute` a dictionary `{"id": myid}` which I think is a lot easier to reason about